service/http.directory/src/test/webapp/symlink/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```

Comment:
Actually, on my computer when I open it, the file only says Hello World. So i am not sure why this occured